apps/text-labeling/app.json

```diff
@@ -0,0 +1,14 @@
+// This file is the descriptor for the app template image-classification
```

Comment:
the comment can be removed especially since it's not `image-classification `

Reply:
Fixed in 93836a8